### ISSUE TITLE
Should these be `--section`?

### ISSUE THREAD

**Issue by lojikil** [NASA-SW-VnV/ikos#199]
https://github.com/NASA-SW-VnV/ikos/blob/a9789d20aa714c6698a37610f5c936406f48d379/analyzer/python/ikos/scan.py#L562

When running ikos locally, I received an error from `llvm-objdump` that the `-e` option was unknown; I checked the source, and modified it thusly: 

```diff
diff --git a/analyzer/python/ikos/scan.py b/analyzer/python/ikos/scan.py
index 6a1c828..144fdd4 100644
--- a/analyzer/python/ikos/scan.py
+++ b/analyzer/python/ikos/scan.py
@@ -559,7 +559,7 @@ def extract_bitcode(exe_path, bc_path):
     # first, extract the llvm bitcode paths
     cmd = [settings.llvm_objdump(), '-s']
     if sys.platform.startswith('darwin'):
-        cmd.append('-section=%s' % DARWIN_SECTION_NAME)
+        cmd.append('--section=%s' % DARWIN_SECTION_NAME)
     elif sys.platform.startswith('linux') or sys.platform.startswith('freebsd'):
         cmd.append('-section=%s' % ELF_SECTION_NAME)
     elif sys.platform.startswith('win'):
```

should this be the case for all `--section` arguments in the code? happy to open a PR with those

**Comment by arthaud** [comment]
That's right. Please submit a PR :)

**Comment by lojikil** [comment]
This is fixed with #200 being merged. thank you @arthaud for the quick turn around!
